src/commerce-handler.js

```diff
@@ -3,6 +3,33 @@ function CommerceHandler(common) {
 }
 
 CommerceHandler.prototype.logCommerceEvent = function(event) {
+    var MBOXNAME = event.CustomFlags['ADOBETARGET.MBOX'];
+    var price = event.ProductAction.TotalAmount || 0;
+    var productSkus = [];
+
+    event.ProductAction.ProductList.forEach(function(product) {
```

Comment:
Maybe we should check if `event.ProductAction` is not null since you're using it in a few places below?